packages/components/src/disabled/README.md

```diff
@@ -8,28 +8,39 @@ Disabled is a component which disables descendant tabbable elements and prevents
 Assuming you have a form component, you can disable all form inputs by wrapping the form with `<Disabled>`.
 
 ```jsx
-const DisableToggleForm = withState( {
-	isDisabled: true,
-} )( ( { isDisabled, setState } ) => {
-	let form = <form><input /></form>;
-
-	if ( isDisabled ) {
-		form = <Disabled>{ form }</Disabled>;
+class ToggleDisable extends React.Component {
```

Comment:
We favor functional components, it should stay as it was. Using Component class with lifecycle methods is more complex, requires more code and might be subject of change in the future. Actually, React 16.3 is a great example how those API can get deprecated.

Reply:
got it!